test/tests/benchmark/utils/time_logger.py

```diff
@@ -0,0 +1,61 @@
+import os
+import time
+import json
+
+"""
+Class to manage time logger file
+"""
+
+class timeLogger(object):
```

Comment:
I will think about that. It is a single file about benchmark case information (perhaps I need to change the class name to make it more clear), about the interval of data collection, every time point that needs to be logged for benchmark event, which is put in the same folder of footprint data collected from RackHD machine (~/benchmark/<timestamp>/<case name>/ in the machine running "python benchmark.py", by default). I prefer to isolate this method in benchmark/ so that it would not confuse other users. Thoughts?